_data/navigation.js

```diff
@@ -248,12 +248,12 @@ module.exports = {
             url: '/docs/listing-services/',
           },
           {
-            title: 'API Reference',
-            url: '/docs/chainlink-framework/',
+            title: 'DevRel Nodes',
+            url: '/docs/any-api-devrel-nodes/',
```

Comment:
Do we need to change the address? I avoid redirects unless they are necessary.

Reply:
we can keep "chainlink-framework" but the url doesn't reflect at all the content of the page

Reply:
Never mind I misread the delta. This change is good, but let's match it to what we decide on for the page title.